lnrpc/rpc.proto

```diff
@@ -908,6 +918,85 @@ message SendToRouteRequest {
     Route route = 4;
 }
 
+message ChannelAcceptRequest {
+    /// The pubkey of the node that wishes to open an inbound channel.
+    bytes node_pubkey = 1;
+
+    /// The hash of the genesis block that the proposed channel resides in.
+    bytes chain_hash = 2;
+
+    /// The pending channel id.
+    bytes pending_chan_id = 3;
+
+    /// The funding amount in satoshis that initiator wishes to use in the channel.
+    int64 funding_amt = 4;
+
+    /// The push amount of the proposed channel in millisatoshis.
+    int64 push_amt = 5;
+
+    /// The dust limit of the initiator's commitment tx.
+    int64 dust_limit = 6;
+
+    /// The maximum amount of coins in millisatoshis that can be pending in this channel.
+    int64 max_value_in_flight = 7;
+
+    /// The minimum amount of satoshis the initiator requires us to have at all times.
+    int64 channel_reserve = 8;
+
+    /// The smallest HTLC in millisatoshis that the initiator will accept.
+    int64 min_htlc = 9;
+
+    /// The initial fee rate that the initiator suggests for both commitment transactions.
+    int64 fee_per_kw = 10;
+
+    /**
+    The number of blocks to use for the relative time lock in the pay-to-self output
+    of both commitment transactions.
+    */
+    uint32 csv_delay = 11;
+
+    /// The total number of incoming HTLC's that the initiator will accept.
+    uint32 max_accepted_htlcs = 12;
+
+    /**
+    The key that should be used on behalf of the sender within the 2-of-2 multi-sig
+    output contained in the funding tx.
+    */
+    bytes funding_key = 13;
+
+    /**
+    The base revocation point for the initiator. Used to derive the revocation key for the
+    commitment transaction.
+    */
+    bytes revocation_point = 14;
+
+    /**
+    The base payment point for the initiator. Used to derive a key that pays directly to the 
+    initiator.
+    */
+    bytes payment_point = 15;
+
+    /**
+    The delay point for the initiator. Used to derive keys used in outputs of the initiator's
+    commitment transaction where they claim funds.
+    */
+    bytes delay_payment_point = 16;
+
+    /// The base point used to derive keys for use in HTLC public key scripts.
+    bytes htlc_point = 17;
+
+    /// The first commitment point for the initiator.
+    bytes first_commitment_point = 18;
+
+    /// A bit-field which the initiator uses to specify proposed channel behavior.
+    uint32 channel_flags = 19;
+}
+
+message ChannelAcceptResponse {
+    /// Whether or not the client accepts the channel.
```

Comment:
If we add a new request/response ID, then the client's implementation will be able to respond to requests out of order, and also ensure we're able to properly dispatch responses to the correct peer. Atm, the funding mgr is single threaded, but that may change in the future as we add more advanced features like internal channel batching.

Reply:
Fixed